flycheck.el

```diff
@@ -8630,6 +8631,16 @@ See URL `http://slim-lang.com'."
           line-end))
   :modes slim-mode)
 
+(flycheck-define-checker slim-lint
+  "A SLIM linter
+
+See URL `https://github.com/sds/slim-lint'."
+  :command ("slim-lint" "--reporter=checkstyle"
+            source-original)
+  :error-parser flycheck-parse-checkstyle
+  :modes slim-mode
+  :next-checkers ((info . slim)))
```

Comment:
@lunaryorn will confirm this, but I would expect `slim-lint` to be the `next-checker` of `slim`, not the other way around.